packages/flutter_tools/lib/src/cache.dart

```diff
@@ -103,6 +103,22 @@ class DevelopmentArtifact {
 ///
 /// This does not provide any artifacts by default. See [FlutterCache] for the default
 /// artifact set.
+///
+/// ## Artifact mirrors
+///
+/// Some environments cannot reach the Google Cloud Storage buckets and CIPD due
+/// to firewall restrictions. A prominent example is China, where the whole
+/// country is behind a firewall, but various corportate environments may
+/// enforce this as well due to company-specific security policies.
```

Comment:
Hmm...We don't need to point out a specific country here. Also, `corportate` -> `corporate`.

```suggestion
/// Some environments cannot reach the Google Cloud Storage buckets and CIPD due
/// to firewall restrictions, various corporate environments may enforce this
/// as well due to company-specific security policies.
```

Reply:
Done. Heads up though, China is already mentioned 5 times in this file. If we don't want to mention it, we should probably clean it up.

Reply:
Haha, a clean up would be good.